book/website/collaboration/hybrid-collab/hybrid-collab-challenges.md

```diff
@@ -0,0 +1,22 @@
+(cl-hybrid-collab-challenges)=
+# Challenges faced during Hybrid Collaboration
```

Comment:
I think you need to add a paragraph at the top which says that the main challenge is creating the same experience and level of incusion for online participants compared to in person participants. 

This then leads nicely to the challenge of social networking, working collaboratively (add this one - about how you will organised online and in person attendes to work together on tasks (synchronous or asynchronously), tools, space.

Reply:
@SaranjeetKaur How about this requested changes?